ci-operator/step-registry/upi/install/libvirt/upi-install-libvirt-commands.sh

```diff
@@ -295,4 +418,5 @@ done
 
 date "+%F %X" > "${SHARED_DIR}/CLUSTER_INSTALL_END_TIME"
 
-touch /tmp/install-complete
+touch ${INSTALL_DIR}/install-complete
+sleep 20m
```

Comment:
Is this sleep for you to debug job runs, or is there additional sleep needed for agent based workflows?

Reply:
As discussed - just debugging for now. will leave this comment thread so I remember to remove.